app.js

```diff
@@ -1,93 +1,109 @@
-if (middleName === 'no') {
+  if (middleName === 'no') {
     alert('Wow, you guessed that my middle name is not Charles. It\'s actually Clifford.');
     console.log(middleName);
-} else {
+  } else {
     alert('You are right, my middle name is actually Clifford.');
     console.log(middleName);
-}
+  }
+} middleName();
 
-var interests = prompt('Can you guess if I\'m interested in coding?');
-if (interests.toLowerCase() === 'yes') {
+function interests() {
+  var interests = prompt('Can you guess if I\'m interested in coding?');
+  if (interests.toLowerCase() === 'yes') {
     alert('Nice, you guessed my primary interest is coding.');
     console.log(interests);
-} else {
-    alert('Actually, my primary interest is coding.')
+  } else {
+    alert('Actually, my primary interest is coding.');
     console.log(interests);
-}
+  }
+} interests();
 
-var interestingFact = prompt('Do you think I attended more than 5 public schools thru High School?');
-if (interestingFact.toLowerCase() === 'yes') {
+function functionName() {
+  var interestingFact = prompt('Do you think I attended more than 5 public schools thru High School?');
+  if (interestingFact.toLowerCase() === 'yes') {
     alert('You are good! I did attend more than 5 schools. I actually attended 6 schools as a \'military brat\' growing up.');
     console.log(interestingFact);
-} else {
+  } else {
     alert('Nope, it was 6 public schools. I was a military brat and moved quite a lot growing up.');
     console.log(interestingFact);
-}
+  }
+} functionName();
 
-var careerGoals = prompt('Do you think my ideal job is that as a Cartographer?');
-if (careerGoals.toLowerCase() === 'yes') {
+function careerGoals() {
+  var careerGoals = prompt('Do you think my ideal job is that as a Cartographer?');
+  if (careerGoals.toLowerCase() === 'yes') {
     alert('You must have a crystal ball! Yep, being a Cartographer is in fact my career goal');
     console.log(careerGoals);
-} else {
-    alert('Well almost... my career goal is in fact to be a Cartographer.')
+  } else {
+    alert('Well almost... my career goal is in fact to be a Cartographer.');
     console.log(careerGoals);
-}
+  }
+} careerGoals();
 
-var focus = prompt('Is Python the computer language you think I\'m focusing my studies?');
-if (focus.toLowerCase() === 'yes') {
+function focus(){
+  var focus = prompt('Is Python the computer language you think I\'m focusing my studies?');
+  if (focus.toLowerCase() === 'yes') {
     alert('Impressive, you are on the money!');
     console.log(focus);
-} else {
+  } else {
     alert('Actually, I\'m focusing my study on Python, as it is apparently the current standard in the GIS industry');
     console.log(focus);
-}
+  }
+} focus();
 
-var failedAttempts = [];
-var computerChoice = Math.floor((Math.random() * 10) + 1);
-var guessNumber = prompt('Guess a number between 1 and 10.');
-for (var i = 1; i < 4; i++) {
-  if (guessNumber === null) {
+//
+function functionName() {
```

Comment:
you have already used the function name functionName (see how confusing this gets without descriptive names!).  What you are doing here is redeclaring the function and overwriting this previous code with new functionality.  Due to js hoisting rules function declarations happen before any of the logic is run, so this functionality below is actually run twice, and your high school question is never run because your high school question functionality is overwritten before it has a chance to run.